src/main/antlr4/org/verdictdb/parser/VerdictSQLParser.g4

```diff
@@ -70,9 +70,11 @@ create_scramble_statement
     : CREATE SCRAMBLE (IF NOT EXISTS)? scrambled_table=table_name FROM original_table=table_name
       (WHERE where=search_condition)?
       (METHOD method=scrambling_method_name)?
-      ((HASHCOLUMN | ON) hash_column=column_name)? 
+      ((HASHCOLUMN | ON) hash_column=column_name)?
+      (STRATIFIED BY stratified_column=column_name_list)?
```

Comment:
I think it could be better if we keep a single syntax for getting column names.
Maybe we can have `((HASHCOLUMN | ON) hash_column=column_name_list)?` for columns required by a scramble. For hash scrambles, we can just take first column in the list (for now).